actindo-grid.html

```diff
@@ -847,7 +864,8 @@
                     '_onItemsChanged(items.*)',
                     '_onSelectionChanged(selectedItems.*)',
                     '_onRemoteChanged(remote, remotePathData, remotePathCount, remoteParams, remoteParamFilter, remoteParamLimit, remoteParamOffset, remoteParamSearch, remoteParamSearchFields, remoteParamSort, itemsPerPage, page, sort, query, filterList)',
```

Comment:
you have to add the properties `remoteParamSerializeOptionals` and `_serializeOptionals.*`  to `_onRemoteChanged` listener, this way you don't have to manually loadDebounce on serialsOptionals change further down.